src/app/qgssnappingwidget.h

```diff
@@ -100,6 +102,10 @@ class APP_EXPORT QgsSnappingWidget : public QWidget
 
     void changeTolerance( double tolerance );
 
+    void changeMinScale( double pMinScale );
```

Comment:
```suggestion
    void setMinimumScale( double minimumScale );
```
arguments are also part of the API due to Python, let's give them nice names
same for `changeMaxScale`

Reply:
no biggie, I see now it's in app.
But better be safe before someone moves this to GUI ;)

Reply:
done